app/models/health/agency_performance.rb

```diff
@@ -143,10 +142,13 @@ def patient_referrals
     # end
 
     def consent_dates
+      # Participation signature was moved into ReleaseForm
       @consent_dates ||= Health::Patient.
-        has_signed_participation_form.
-        # where(hpf_t[:signature_on].between(@range)).
-        pluck(:patient_id, hpf_t[:signature_on].to_sql).to_h
+        joins(:release_forms).
+        pluck(:patient_id, hrf_t[:participation_signature_on].to_sql).to_h
```

Comment:
I _think_ you'll want an order by on here.  Also, I think you can `where(hrf_t[:participation_signature_on].not_eq(nil))`.  The `to_h` on the end will be problematic without an order clause if a patient has more than one release form.

Reply:
I'll replace this with a group -- they only need to sign the participation agreement once, so we can just use their earliest one.